### ISSUE TITLE
Wrong convertion metrics when converting

### ISSUE THREAD

**Issue by chicocode** [danielpalme/OpenCoverToCoberturaConverter#15]
The summary of the OpenCover file is 
```
<Summary numSequencePoints="149138"
    visitedSequencePoints="10557" 
    numBranchPoints="42528" 
    visitedBranchPoints="4549"
    sequenceCoverage="7.08" 
    branchCoverage="10.70"
    maxCyclomaticComplexity="211" 
    minCyclomaticComplexity="0" 
    visitedClasses="652" 
    numClasses="3770" 
    visitedMethods="4301" 
    numMethods="32966" />
```
When I run the following command
`&$PSScriptRoot\Packages\OpenCoverToCoberturaConverter.0.2.6.0\tools\OpenCoverToCoberturaConverter.exe -input:"$PSScriptRoot\cake\coverage-result.xml" -output:"$PSScriptRoot\cake\Cobertura.xml" -includeGettersSetters:"true"`

It gives a file with the summary below
```
<coverage line-rate="0.066887933562707" 
    branch-rate="0.0242214532871972" 
    lines-covered="9351" 
    lines-valid="139801" 
    branches-covered="210" 
    branches-valid="8670" 
    complexity="0" 
    version="0" 
    timestamp="1508343734">
```
Seems like the conversion wasn't properly done.
Am I doing something wrong?

**Comment by danielpalme** [comment]
I wrote this little tool 4 years ago.
As far as I remember, the numbers in the generated Cobertura file match the number of generated <line /> elements in the file. Those can differ from the numbers in the OpenCover file.

I did a quick test and the number of <line /> elements are consistent with the metrics in the summary.

Since I don't remember every detail of the conversion, you might want to create a concrete minimum sample OpenCover file and verify the results manually.
